libopae/plugins/xfpga/metrics/metrics_utils.c

```diff
@@ -423,12 +414,10 @@ fpga_result enum_perf_counter_metrics(fpga_metric_vector *vector,
 		return FPGA_INVALID_PARAM;
 	}
 
-	snprintf_s_ss(sysfs_path, sizeof(sysfs_path), "%s/%s", sysfspath, PERF);
-
-	glob_t pglob;
-	int gres = glob(sysfs_path, GLOB_NOSORT, NULL, &pglob);
+	int gres = glob(sysfspath, GLOB_NOSORT, NULL, &pglob);
 	if ((gres) || (1 != pglob.gl_pathc)) {
 		globfree(&pglob);
```

Comment:
Same thing here, should we call globfree before we exit the function?

Reply:
pglob is freed in line 427

Reply:
fixed